restapi.go

```diff
@@ -2545,6 +2545,62 @@ func (s *Session) ApplicationCommands(appID, guildID string) (cmd []*Application
 	return
 }
 
+// GuildApplicationCommandsPermissions receives all permissions for application commands that have any permissions
+// appID       : The application ID
+// guildID     : Guild ID to retrieve all guild-specific permissions for application commands in the guild, both global and guild specific
+func (s *Session) GuildApplicationCommandsPermissions(appID, guildID string) (permissions []*GuildApplicationCommandPermissions) {
```

Comment:
Is there a specific reason why this function returns `[]*GuildApplicationCommandPermissions`, while `ApplicationCommandPermissions` returns a non-pointer `GuildApplicationCommandPermissions`, and `ApplicationCommandPermissionsBatchEdit` takes a slice of non-pointer `[]GuildApplicationCommandPermissions`? It seems like we'd most likely want to keep some consistency here.

Reply:
Good point, wasn't very focused on it. I've changed it to use pointer types where possible for structs